sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -2557,12 +2557,7 @@ def hpat_pandas_series_add(self, other, level=None, fill_value=None, axis=0):
     # specializations for numeric series only
     if not operands_are_series:
         def _series_add_scalar_impl(self, other, level=None, fill_value=None, axis=0):
-            fill_value_is_nan = False
-            if fill_value is None:
-                fill_value = numpy.nan
-            if not fill_value_is_none == True:  # noqa
-                fill_value_is_nan = numpy.isnan(fill_value)
-            if not (fill_value_is_nan or fill_value_is_none == True):  # noqa
+            if (fill_value is not None and not numpy.isnan(fill_value)):
```

Comment:
Don't you like the shorter form?
```suggestion
            if not (fill_value is None or numpy.isnan(fill_value)):
```